app-admin/u-boot-menu/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=u-boot-menu
+PKGSEC=misc
+PKGDES="u-boot menu update"
```

Comment:
Please improve this description, tell the user what it does (and capitalise the first letter).